larray_editor/editor.py

```diff
@@ -219,7 +403,9 @@ def void_formatter(array, *args, **kwargs):
             arrays = [k for k, v in self.data.items() if self._display_in_grid(k, v)]
             self.add_list_items(arrays)
         self._listwidget.setCurrentRow(0)
-        return True
+
+    def type_editor(self, readonly):
+        return "Session Viewer" if readonly else "Session Editor"
```

Comment:
Maybe it's time to use a single name (Session Editor) whether readonly or not (we add the "readonly" qualifier to the title anyway) so that we can use a simple class attribute instead of a method?